lib/reporting/reports/sales_tax/sales_tax_totals.rb

```diff
@@ -0,0 +1,152 @@
+# frozen_string_literal: true
+
+module Reporting
+  module Reports
+    module SalesTax
+      class SalesTaxTotals < Base
```

Comment:
It looks like we should make this class shorter. Otherwise we should add it to the rubocop todo list so that other devs can use rubocop without seeing known issue they didn't cause.